.github/workflows/check-full.yaml

```diff
@@ -0,0 +1,62 @@
+# Workflow derived from https://github.com/r-lib/actions/tree/master/examples
+# Need help debugging build failures? Start at https://github.com/r-lib/actions#where-to-find-help
+#
+# NOTE: This workflow is overkill for most R packages and
+# check-standard.yaml is likely a better choice.
+# usethis::use_github_action("check-standard") will install it.
+on:
+  push:
+    branches: [main, master]
+  pull_request:
+    branches: [main, master]
+  schedule:
+    - cron:  '30 12 * * 3,6'
+  repository_dispatch:
+    types: [trigger-check-workflow]
+  workflow_dispatch:
+
+
+name: R-CMD-check
+
+jobs:
+  R-CMD-check:
+    runs-on: ${{ matrix.config.os }}
+
+    name: ${{ matrix.config.os }} (${{ matrix.config.r }})
+
+    strategy:
+      fail-fast: false
+      matrix:
+        config:
+          - {os: windows-latest, r: 'devel'}
+          - {os: windows-latest, r: 'release'}
+          - {os: windows-latest, r: 'oldrel'}
+          # - {os: macOS-latest, r: 'devel'}
```

Comment:
why is this commented out?

Reply:
Currently there is an overall issue on macOS devel, it seems that the R installation failed hence the check is not passed (unrelated with this specific package, false negative ). I removed the comment, see the result https://github.com/dieghernan/federalregister/actions/runs/13828248584